### ISSUE TITLE
Bug when graph=adj flag specified

### ISSUE THREAD

**Issue by emreerhan** [bcgsc/abyss#142]
Hi again,

It seems like when the graph=adj flag is specified for abyss-pe, the name-2.adj file is still made as a dot file. However, the name-2.adj1 file is made correctly as an adj file. If the flag isn't specified, I think these two files are exactly the same. I tried converting the name-2.adj1 file to dot using abyss-todot, and tried to diff the files to see if they are indeed the same. This is the result:
```
~> diff KC2Z01229negative-2.dot1 KC2Z01229negative-2.adj
1a2,3
> graph [k=38]
> edge [d=-37]
```

This isnt a very big deal, but I thought it would be worth mentioning.

**Comment by benvvalk** [comment]
Noted. Thanks for taking the time to report it!

**Comment by sjackman** [comment]
Looks like the bug is at https://github.com/bcgsc/abyss/blob/master/bin/abyss-pe#L518
Untested fix:
```diff
-	MergeContigs $(mcopt) -g $*-2.$g -o $*-2.fa $^
+	MergeContigs $v --$g $(mcopt) -g $*-2.$g -o $*-2.fa $^
```

**Comment by benvvalk** [comment]
Thanks, Shaun!

**Comment by sjackman** [comment]
This bug exists in the current stable release of ABySS. It's fixed in the develop branch. The fix will be included in the upcoming release of ABySS 2.0.3.
https://github.com/bcgsc/abyss/blob/b75461116ffc2e7b9c0034fcf838fc4afe9912d9/bin/abyss-pe#L518
